cli/test.sh

```diff
@@ -35,13 +38,31 @@ node build/index.js genProofs -x 0xf204a4Ef082f5c04bB89F7D5E6568B796096735a \
     -f proofs.json
 
 node build/index.js proveOnChain \
-    -x 0xf204a4Ef082f5c04bB89F7D5E6568B796096735a \
     -o 0 \
-    -q 0xEcFcaB0A285d3380E488A39B4BB21e777f8A4EaC \
     -f proofs.json
 
 node build/index.js verify \
-    -x 0xf204a4Ef082f5c04bB89F7D5E6568B796096735a \
     -o 0 \
     -t tally.json \
-    -q 0xEcFcaB0A285d3380E488A39B4BB21e777f8A4EaC
+
+node build/index.js mergeMessages -o 1 && \
+node build/index.js mergeSignups -o 1 
+rm -f tally.json proofs.json && \
```

Comment:
Sorry for the merge conflict but now proofs can span across multiple files for speed improvements

Reply:
I have rebased the v1 code. No conflicts now.